scripts/datasets/DET.py

```diff
@@ -0,0 +1,203 @@
+"""Prepare DET(Object detection) datasets"""
```

Comment:
Please provide more details, like `this script is used to prepare DET dataset for tracking`, `Which year of ImageNet used?`, etc.